tests/unit/core/accessor/test_datetime.py

```diff
@@ -16,7 +16,7 @@
         (lambda s: s.dt.year, 'EXTRACT(year FROM "TIMESTAMP")'),
         (lambda s: s.dt.quarter, 'EXTRACT(quarter FROM "TIMESTAMP")'),
         (lambda s: s.dt.month, 'EXTRACT(month FROM "TIMESTAMP")'),
-        (lambda s: s.dt.week, 'EXTRACT(week FROM "TIMESTAMP")'),
+        # (lambda s: s.dt.week, 'EXTRACT(week FROM "TIMESTAMP")'),  # Not supported in pandas 2.
```

Comment:
BREAKING

Reply:
`s.dt.week` should be replaced with `dt.isocalendar().week` in pandas 2